src/games/supported/AirRaid.cpp

```diff
@@ -15,8 +15,10 @@
 
 
 AirRaidSettings::AirRaidSettings() {
-
-    reset();
+    m_reward   = 0;
```

Comment:
Setting the values here (and in reset() below) breaks the previous semantics of reset(). Do you think we can combine the two back together?